src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List<? extends Machine> get();
```

Comment:
The `MachineProducer` interface should be parameterized with a bounded type parameter to ensure that only `Machine` or its subtypes can be used. For example, `MachineProducer<T extends Machine>`. This will prevent the interface from being implemented with unrelated types such as `String`.